.gitignore

```diff
@@ -2,9 +2,10 @@
 /build/*
 /dist/*
 /RosettaSciIO.egg-info/*
+home/
 *__pycache__*
 *test_compilers.obj
 *pyd
 .spyproject/*
-rsciio/bruker/unbcf_fast.c
-.idea
\ No newline at end of file
+rsciio/bruker/unbcf_fast.c*
+.idea
```

Comment:
How do these changes filter the `unbcf_fast*.so` file?

Reply:
Right, this is a directory used in PyCharm projects. Should I add it in a separate commit?

Reply:
No this is fine, I was asking about the `unbcf_fast*.so` file and this is discussed below.